src/meetingreadinesschecker/DefaultMeetingReadinessChecker.ts

```diff
@@ -162,7 +168,7 @@ export default class DefaultMeetingReadinessChecker implements MeetingReadinessC
   }
 
   async checkCameraResolution(
-    videoInputDevice: MediaDeviceInfo,
+    videoInputDevice: MediaDeviceInfo | string,
```

Comment:
Consider whether changing this to `Device` makes sense — after all, in this function you're constructing constraints and then getting a stream, and the other kinds of `Device` are exactly those two things!

Reply:
I decided to stick with just MediaDeviceInfo | string since this API is to specifically check for a camera device.

Reply:
When we design this API, I do not think we have that use cases in mind (The name camera resolution means specifically to a device). What you mentioned is a valid use case but I think we may want to create a different API for it.

Reply:
That's fair!